Evolution.md

```diff
@@ -22,3 +22,44 @@ When the scheduled review period arrives, the review manager will post the propo
 
 After the review has completed, the Foundation Workgroup will make a decision on the proposal. The review manager is responsible for determining consensus among the Foundation Workgroup members, then reporting their decision to the proposal authors and forums. The review manager will update the proposal's state in the repository to reflect that decision.
 
+### Review Announcement
+
+(Credit: This section is adapted from [Swift-Evolution's announcement template](https://github.com/apple/swift-evolution/blob/main/process.md#review-announcement))
+
+When a proposal enters review, a new topic will be posted to the [Foundation project of the Swift forums](https://forums.swift.org/c/related-projects/foundation/) using the following template:
+
+---
+Hello Swift community,
+
+The review of [\<\<PROPOSAL NAME>>]\(\<\<LINK TO PROPOSAL>>) begins now and runs through \<\<REVIEW END DATE>>
+
+Reviews are an important part of the Swift-Foundation evolution process. All review feedback should be either on this forum thread or, if you would like to keep your feedback private, directly to me as the review manager by \<\<CONTACT METHOD>>. When contacting the review manager directly, please include proposal name in the subject line.
+
+
+##### Trying it out
+
+If you'd like to try this proposal out, you can check out \<\<LINK TO IMPLEMENTATION>>.
```

Comment:
Does this imply that all evolution needs to have implementation to go with it?

Reply:
I think so, yes